types/office-js-preview/index.d.ts

```diff
@@ -5307,6 +5307,13 @@ declare namespace Office {
          *
          * This method returns a promise. Use it with Excel, PowerPoint, and Word add-ins.
          *
+         * If a user grants access to a device capability for the first time, the promise resolves with `true`.
+         * You must then reload the add-in in the browser before you can run code that uses the device capability.
```

Comment:
Can we get rid of the phrase "in the browser"? I don't think it adds anything and I think it might be misleading.

Reply:
Removed.